templates/patterns/search_block/pattern-search-block.html.twig

```diff
@@ -0,0 +1,26 @@
+{#
+/**
+ * @file
+ * Default theme implementation for a 'form' element.
+ */
+#}
+{% if button_icon %}
+  {% set _icon = {
+      name: button_icon,
+      path: bcl_icon_path
+    }
+  %}
+{% endif %}
+<form{{ attributes.addClass('search-form', 'search-block-form', additional_classes) }} action="{{ url }}">
```

Comment:
create a attributes array
```
{% if attributes is empty %}
  {% set attributes = create_attribute() %}
{% endif %}
```
start adding the classes, action to it, this way if the pattern sends more then the scenario is covered